lib/install/examples/stimulus/application.js

```diff
@@ -0,0 +1,6 @@
+import { Application } from "stimulus"
+import { definitionsFromContext } from "stimulus/webpack-helpers"
+
+const application = Application.start()
+const context = require.context("./controllers", true, /.js$/)
```

Comment:
Does `"./controllers"` work? That's not the relative path to the controllers directory. When I tested, `"controllers"` worked as expected since webpacker adds all root `app/javascript/` directories to the load path.

Reply:
Thanks for pointing this out. I uninstalled the old webpacker and now, it's working for me 🤷‍♂️